scripts/deploy.sh

```diff
@@ -14,10 +14,23 @@ if [ "$(whoami)" != "zds" ]; then
 fi
 
 if [ "$#" -ne 1 ]; then
-	echo "Usage: $0 <tag name>" >&2
+	echo "Usage: $0 <tag name> <option>" >&2
+	echo "Options:"
+	echo "  --prod (default)"
+	echo "  --beta"
 	exit 1
 fi
 
+# paths
+ENV_PATH="/opt/zds/zdsenv/"
+REPO_PATH="/opt/zds/zds-site/"
+if [[ $1 == "--beta" ]]
+then
+    ENV_PATH="/opt/zdsenv/"
+    REPO_PATH="/opt/zdsenv/ZesteDeSavoir/"
+
+fi
+
 read -p "Did you run specific tasks for this version as described in update.md? [y/N] " -r
```

Comment:
Je comprends, et c'est super pénible. Par contre tant qu'on aura des données à corriger, ce sera un peu inévitable.

Normalement à la prochaine MEP on va corriger les doublons de souscriptions et éviter que ça se reproduise, et stripper les espaces avant et après les tags et supprimer les doublons et éviter que ça se reproduise.